### ISSUE TITLE
Dataset loader

### ISSUE THREAD

**Issue by Blubberli** [Blubberli/ambiguous-alpaca#29]
load a dataset with pandas (we have to decide on one or two formats that we think the dataset would look like)
maybe
sentence SEP phrase SEP label
or just
phrase SEP label

then we should use the pytorch dataset utility to return a Dataloader that we can extract batches from...
